recipes/lantz/meta.yaml

```diff
@@ -0,0 +1,111 @@
+{% set name = "lantz" %}
+{% set version = "0.5.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # lantz-core
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}-core/lantz-core-{{ version }}.tar.gz
+    sha256: 9ad1b74c1549fb31db492b3cb836bf710f4b7b9a0c729e0a908c7eec21db0afb
+    folder: core
+  # lantz-qt
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}-qt/lantz-qt-{{ version }}.tar.gz
+    sha256: 3fd56cdce6039f5ca74b32fed059c4ebfaf0aa713e28a64ce5707881469de56e
+    folder: qt
+  # lantz-sims
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}-sims/lantz-sims-0.5.2.tar.gz
+    sha256: 5e538f7a5b40ffeb16795df0a053dc0106d35b1e4f3fbdfad8995123efced431
+    folder: sims
+  # lantz-drivers
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}-drivers/lantz-drivers-0.5.2.tar.gz
+    sha256: 8a8de249f07fda0ae008f55446f8e2964ff793fe6bde07be492435823bbf1023
+    folder: drivers
+  # lantz-ino
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}-ino/lantz-ino-0.5.2.tar.gz
+    sha256: b5a0d9540180cf93b2184d080c0372308da1f98ee881d1dd10cabfa93c617735
+    folder: ino
+  # lantz-dev
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}dev/lantzdev-0.5.2.tar.gz
+    sha256: d13e828bccd57fb7e29e417dbf78626d2ea3dd3ecdd429440830205788376eb1
+    folder: .
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - lantz = lantz.__main__:main
+    - lantz-config = lantz.core.__main__:config
+    - lantz-qtdemo = lantz.qt.__main__:main
+    - lantz-sims = lantz.sims.__main__:main
+    - lantz-ino = lantz.ino.__main__:main
+  script:
+    - {{ PYTHON }} -m pip install https://pypi.io/packages/source/P/PySignal/PySignal-1.1.1.tar.gz
```

Comment:
Please make a separate conda package for `PySignal`

Reply:
Please see PR https://github.com/conda-forge/staged-recipes/pull/12557 for this.

Reply:
Can you do the same for the other deps?

Reply:
The remaining deps in the install are all no longer actively maintained. They were written for the original version of this project and are kept for legacy reasons. I don't recommend uploading them as separate packages.

Reply:
Even if so, it's not our policy to package different packages in the same conda package. Please create separate conda packages.

Reply:
fixed